kumascript/src/live-sample.js

```diff
@@ -0,0 +1,145 @@
+const ejs = require("ejs");
+
+const Parser = require("./parser.js");
+const { normalizeMacroName } = require("./render.js");
+const { HTMLTool, slugify, safeDecodeURIComponent } = require("./api/util.js");
+
+const LIVE_SAMPLE_HTML = `
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <style type="text/css">
+            body {
+              padding: 0;
+              margin: 0;
+            }
+            
+            svg:not(:root) {
+              display: block;
+            }
+            
+            .playable-code {
+              background-color: #f4f7f8;
+              border: none;
+              border-left: 6px solid #558abb;
+              border-width: medium medium medium 6px;
+              color: #4d4e53;
+              height: 100px;
+              width: 90%;
+              padding: 10px 10px 0;
+            }
+            
+            .playable-canvas {
+              border: 1px solid #4d4e53;
+              border-radius: 2px;
+            }
+            
+            .playable-buttons {
+              text-align: right;
+              width: 90%;
+              padding: 5px 10px 5px 26px;
+            }
+        </style>        
+        <% if (css) { %>
+        <style type="text/css">
+            <%- css %>
+        </style>
+        <% } %>
+        <title><%- sampleTitle %></title>
+    </head>
+    <body>
+        <% if (html) { %>
+            <%- html %>
+        <% } %>
+        <% if (js) { %>
+            <script>
+                <%- js %>
+            </script>
+        <% } %>
+    </body>
+</html>`;
```

Comment:
```suggestion
</html>`.trim();
```
otherwise every live sample will start with a newline.